src/checkers/inference/DefaultInferenceResult.java

```diff
@@ -0,0 +1,14 @@
+package checkers.inference;
+
+import checkers.inference.model.Constraint;
+
+import javax.lang.model.element.AnnotationMirror;
+import java.util.Collection;
+import java.util.Map;
+
+public class DefaultInferenceResult extends BaseInferenceResult {
+    public DefaultInferenceResult(Map<Integer, AnnotationMirror> annotationResults,
```

Comment:
This class is exactly the same as its super class.

Why we need this class?

Reply:
Maybe we can directly instantiate `BaseInferenceResult` in place of `DefaultInferenceResult`?

Reply:
Removed `BaseInferenceResult`. Use only `DefaultInferenceResult`